### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/18/day17.html?fbclid=IwAR0dAlc5Y22RSPcaN_afKu4MSfoguZsXmmU7C2pOWJ_RsnucAtZ_sBij_0g_aem_AcHef0Lh2MBEyxu4aFBtOVwo0FRBzopMv_E7mH2tzSRgkVpdDqrIQJT3HEbpkKpAuSk

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#25] [Comments]
# https://mnfienen.github.io/talulat/2023/08/18/day17.html?fbclid=IwAR0dAlc5Y22RSPcaN_afKu4MSfoguZsXmmU7C2pOWJ_RsnucAtZ_sBij_0g_aem_AcHef0Lh2MBEyxu4aFBtOVwo0FRBzopMv_E7mH2tzSRgkVpdDqrIQJT3HEbpkKpAuSk

Day 17 - Some days are just hard and bike parts can shred

https://mnfienen.github.io/talulat/2023/08/18/day17.html?fbclid=IwAR0dAlc5Y22RSPcaN_afKu4MSfoguZsXmmU7C2pOWJ_RsnucAtZ_sBij_0g_aem_AcHef0Lh2MBEyxu4aFBtOVwo0FRBzopMv_E7mH2tzSRgkVpdDqrIQJT3HEbpkKpAuSk

**Comment by SmittysGitHub** [comment]
When the going gets tough, you just keep going!!
